bot/cogs/hacktoberstats.py

```diff
@@ -0,0 +1,184 @@
+import re
+import typing
+from collections import Counter
+from datetime import datetime
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+
+class Stats:
+    def __init__(self, bot):
+        self.bot = bot
+        self.channelID = (496432022961520650)  # Hardcode #event-hacktoberfest channel ID
+
+    @commands.command(
+        name="stats",
+        aliases=["getstats", "userstats"],
+        brief="Get a user's Hacktoberfest contribution stats",
+    )
+    async def getstats(self, ctx, *args):
+        """
+        Query GitHub's API for PRs created by a GitHub user during the month of October that 
+        do not have an 'invalid' tag
+
+        For example:
+            !getstats heavysaturn
+
+        If a valid username is provided, an embed is generated and posted to the channel
+
+        Otherwise, post a helpful error message
+
+        The first input argument is treated as the username, any additional inputs are discarded
+        """
+        username = args[0]
```

Comment:
Why not accept a single argument instead?
```py
commands.command(...)
async def getstates(self, ctx, username: str):
    ...
```
`username` will be equal to the first argument to the function (e.g. `!getstats XYZ` lets username be `"XYZ"`)

Reply:
The intent was to guard against too many input arguments but it looks like d.py already handles this for us.

Reply:
:+1: